README.md

```diff
@@ -11,12 +11,17 @@ assets for use in probo builds.
 
 ## Usage
 
- 1. Create a bucket to hold assets.
- 2. Create a token that can be used to post assets into that bucket.
- 3. Upload an asset into a bucket with the upload token.
+ 1. Add Amazon S3 credentials.
```

Comment:
The whole idea of creating a plugin system is that the S3 storage option is an optional change, this should be a new section of the readme not changing the main usage docs.